packages/api/portfolioDrafts/application/getApplicationStep.ts

```diff
@@ -1,14 +1,23 @@
 import { GetCommand } from "@aws-sdk/lib-dynamodb";
-import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { APIGatewayProxyResult, Context } from "aws-lambda";
 import { ApplicationStep } from "../../models/ApplicationStep";
 import { APPLICATION_STEP } from "../../models/PortfolioDraft";
 import { dynamodbDocumentClient as client } from "../../utils/aws-sdk/dynamodb";
-import { DATABASE_ERROR, NO_SUCH_APPLICATION_STEP, PATH_PARAMETER_REQUIRED_BUT_MISSING } from "../../utils/errors";
-import { ApiSuccessResponse, ErrorStatusCode, OtherErrorResponse, SuccessStatusCode } from "../../utils/response";
-import { isPathParameterPresent, isValidUuidV4 } from "../../utils/validation";
+import { DATABASE_ERROR, NO_SUCH_APPLICATION_STEP } from "../../utils/errors";
+import {
+  ApiSuccessResponse,
+  ErrorStatusCode,
+  OtherErrorResponse,
+  SetupError,
+  SuccessStatusCode,
+} from "../../utils/response";
+import middy from "@middy/core";
+import cors from "@middy/http-cors";
+import { ApiGatewayEventParsed } from "../../utils/eventHandlingTool";
+import { shapeValidationForPostRequest } from "../../utils/requestValidation";
 
 // Note that API spec calls for 400 and not 404
-export const NO_SUCH_PORTFOLIO_DRAFT = new OtherErrorResponse(
+export const NO_SUCH_PORTFOLIO_DRAFT_FOUND = new OtherErrorResponse(
   "The given Portfolio Draft does not exist",
   ErrorStatusCode.BAD_REQUEST
 );
```

Comment:
We should use `NO_SUCH_PORTFOLIO_DRAFT` from errors.ts instead here.

Reply:
I think I just pushed that change. I removed `NO_SUCH_PORTFOLIO_DRAFT_FOUND`.